lib/needle.js

```diff
@@ -763,7 +777,16 @@ Needle.prototype.send_request = function(count, method, uri, config, post_data,
     request.end();
   }
 
-  out.abort = function() { request.abort() }; // easier access
+  // Manage the abort signal
+  if ( signal ) {
+    if ( signal.aborted === true ) {
```

Comment:
Please try to follow the syntax used in the rest of the code (no spaces between parenthesis).